docs/about/authors.rst

```diff
@@ -0,0 +1,26 @@
+*******
+Authors
+*******
+
+**challengeutils** would not be possible without the evergrowing
+efforts of:
+
+- `Thomas Yu`_
```

Comment:
No need to include all these authors

Reply:
but... they contributed a commit... :>

Reply:
But it would be high effort to continuously add authors that contribute.  It would be easier to link to that GitHub page that automatically tracks the contributors who aren't main authors.